udata/frontend/markdown.py

```diff
@@ -39,20 +42,42 @@ def nofollow_callback(attrs, new=False):
     That callback is not splitted in order to parse the URL only once.
     """
 
+    print(f'\n>>>>> nofollow_callback / attrs : \n{attrs}')
+    # print(f">>>>> nofollow_callback / current_app.config['SERVER_NAME'] : \n{current_app.config['SERVER_NAME']}")
+
     if (None, u"href") not in attrs:
+        print(f'>>>>> nofollow_callback / attrs X : \n{attrs}')
         return attrs
+
     parsed_url = urlparse(attrs[(None, 'href')])
+    print(f'>>>>> nofollow_callback / parsed_url : \n{parsed_url}')
+
+    if parsed_url.scheme == u"mailto" :
+        print(f'>>>>> nofollow_callback / mailto ...')
+        scheme = 'mailto'
```

Comment:
What is this block of code for?
When an email is parsed we want to turn it to plain text and to avoid mailto.

Reply:
oh I though we wanted to keep the ability to write to someone... I'll change that to turn it to plain text

Reply:
Has this been taken care of?